src/lib/modules/Loan/actions.js

```diff
@@ -60,12 +60,7 @@ export const performLoanAction = (
         type: ACTION_SUCCESS,
         payload: response.data,
       });
-
-      dispatch({
-        type: DETAILS_SUCCESS,
-        payload: response.data,
-      });
-
+      dispatch(fetchLoanDetails(response.data.pid));
```

Comment:
```suggestion
      dispatch(fetchLoanDetails(response.data.medata.pid));
```
?

I would really love to remove that extra `pid` at the root level